packages/blade/src/components/Amount/Amount.stories.tsx

```diff
@@ -217,3 +223,98 @@ StrikeThrough.args = {
   isStrikethrough: true,
 };
 StrikeThrough.storyName = 'Strike Through';
+
+// TODO: Replace below with i18nify getDefaultLocales API
```

Comment:
is it yet to be implemented?

Reply:
Yes this is supposed to be implemented in the upcoming sprints. As this is just for storybook, we can update this once we have it ready in `i18nify`.